src/services/audius-api-client/AudiusAPIClient.ts

```diff
@@ -60,6 +110,156 @@ class AudiusAPIClient {
     return adapted
   }
 
```

Comment:
Not for this pr, but we're gonna need to break this file up sooner rather than later before it's too late

Reply:
Yep +1